static/js/biomedical/landing/biomedical_search_box.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Search Box for the Biomedical DC landing page
+ */
+
+import React, { useEffect, useRef, useState } from "react";
+import { Input, InputGroup } from "reactstrap";
+
+interface BiomedicalSearchProps {
+  // Whether to allow searching for an empty string
+  allowEmptySearch?: boolean;
```

Comment:
curious, is there a case where we allow empty search?

Reply:
I originally kept it to match the specs of the NL search bar on main DC, but removed it since I can't think of a case where we'd allow empty search.